### PR TITLE
cease compiling unused lib files as per psi4/psi4#525

### PR DESCRIPTION
**Pull request by loriab**
## Description
Hopefully reduce compile time

## Todos
Notable points that this PR has either accomplished or will accomplish.
* **Developer Interest**
  - [x] Removed all the lib files identified as unused except those in libdpd

## Status
- [x]  Ready to go

### CODE REVIEW

**Comment by amjames** [comment]
@lothain commented via email and it seems to have been lost in cyberspace:

>I know that Rollin's calculation ran with both codes, but I do not believe we have confirmed that the disk space requirements of cctranssort are reasonable

I was able to run the calculations that his comment refers to, both using cctransort and using ccsort/transqt2. The disk requirements for the transformation/sorting integrals were identical, so I believe we are safe in moving ccsort/transqt2 to the attic.

File: psi4/src/psi4/libmints/CMakeLists.txt [hunk 1]
```diff
@@ -1,70 +1,70 @@
...
+                 intvector.cc
+                 potential.cc
+                 mintshelper.cc
+                 coordentry.cc
+                 kinetic.cc
+                 tracelessquadrupole.cc
+                 pseudospectral.cc
+                 integral.cc
+                 matrix.cc
+                 #svd.cc
```
Comment (jturney) [inline psi4/src/psi4/libmints/CMakeLists.txt]: It's save to delete ```svd.cc``` completely.

**Comment by andysim** [comment]
LGTM, but we should consult others first.  Perhaps this is a good one to moot on our next monthly call.

**Comment by dgasmith** [comment]
Just a general git point: This one and #534 are tied to the same branch history so neither should be rebased. Its one of the downsides of a rebase team policy. Merges are fine.

**Comment by dgasmith** [comment]
This is tied into #534 now. Closing.

**Pull request closed without merging**